Content.Shared/Damage/Container/DamageContainerPrototype.cs

```diff
@@ -9,7 +9,7 @@
 namespace Content.Shared.Damage.Container
 {
     /// <summary>
-    ///     Prototype for the DamageContainer class.
+    ///     Prototype for DamageableComponents.
```

Comment:
what is this doc meant to convey?

Reply:
I've made the docs here, and in DamageGroupPrototype somewhat more descriptive. I could also just remove them though, it should be pretty self explanatory. The above is now:

```
<summary>
    A damage container which can be used to specify support for various damage types.
</summary>
<remarks>
    This is effectively just a list of damage types that can be specified in YAML files using both damage types
    and damage groups. Currently this is only used to specify what damage types a <see
    cref="Components.DamageableComponent"/> should support.
</remarks>
```